cases/services/tag_enricher.py

```
@@ -0,0 +1,1134 @@
+                )
+
+        tags = classify_case_rules(case)
+
+        if self.use_llm and len(tags) < 5:
+            try:
+                llm_tags = self._classify_with_llm(case)
+                if llm_tags:
+                    all_tags = list(dict.fromkeys(tags + llm_tags))
+                else:
+                    all_tags = tags
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)